src/test/java/org/fundacionjala/trello/stepdefs/StepDefs.java

```diff
@@ -11,10 +12,12 @@
 public final class StepDefs {
 
     private Context context;
+    private RequestManager requestManager;
     private LoginPage loginPage;
 
-    public StepDefs(final Context context) {
+    public StepDefs(final Context context, final RequestManager requestManager) {
```

Comment:
Just have a question here, why is it necessary to include a request manager?

Reply:
It is because, we need to authenticate the given user to connect to Trello and send requests by API.
Without that, we should specify with a step in the gherkin files that we will authenticate by API.

Reply:
Oh, but wouldn't it be good to specify that on the Gherkin files?